alica_tests/src/test/etc/plans/behaviours/TestInheritBlackboardBehaviour.beh

```diff
@@ -1,24 +1,15 @@
 {
+  "blackboard": [],
   "comment": "",
   "deferring": 0,
   "eventDriven": false,
   "frequency": 30,
   "id": 831400441334251600,
+  "inheritBlackboard": true,
   "name": "TestInheritBlackboardBehaviour",
   "postCondition": null,
   "preCondition": null,
   "relativeDirectory": "Behaviour",
-  "requiresParameters": true,
   "runtimeCondition": null,
-  "variables": [],
-  "inheritBlackboard" : true,
-  "blackboard" : [
-    {
-      "key" : "behaviorKey",
-      "type" : "std::any",
-      "access" : "protected",
-      "defaultValue" : null,
-      "comment" : "This is a blackboard entry for testing"
-    }
-  ]
```

Comment:
we're removing BB because we should be inheriting it?

Reply:
a plan/behaviour which inherits a blackboard cannot define a blackboard key, but somehow this .beh file violated this. So I had to remove the blackboard key.